config/checkstyle/suppressions.xml

```diff
@@ -9,5 +9,6 @@
 	<suppress checks="MultipleStringLiterals" files="src/test/*" />
 	<suppress checks="MultipleStringLiterals" files="src/integrationTest/*" />
 	<suppress checks="InnerAssignment" files="src/test/*" />
-	<suppress checks="InnerAssignment" files="src/integrationTest/*" />
+    <suppress checks="InnerAssignment" files="src/integrationTest/*" />
```

Comment:
Looks like indentation is not aligned here!

Reply:
Ah that's because suppressions.xml is indented with tabs. I have Eclipse set to expand tabs to spaces, which I forgot to disable while editing that file. I'll go ahead and fix that. Perhaps it would be better to change suppressions.xml to be indented with spaces? I could update that as well if you think it's a good idea.

Reply:
I will change the indentation to spaces.